federatedscope/autotune/algos.py

```diff
@@ -372,6 +372,7 @@ def _setup(self):
                         for j in range(self._cfg.hpo.table.num - 1))
             arms['arm0'] = dict(
                 (k, v) for k, v in trial_cfg.items() if k in arms['arm1'])
+            os.makedirs(self._cfg.hpo.working_folder, exist_ok=True)
```

Comment:
I am not sure whether this is an appropriate position to do this thing.

Reply:
Updated.